api/views/workflow.py

```diff
@@ -168,3 +170,96 @@ def get_queryset(self):
             ).distinct('id')
         else:
             return RawData.objects.all()
+
+
+class V1DataView(views.APIView):
+    """View allowing sources to post SHARE v1 formatted metadata directly to the SHARE Dataset.
+
+    ## Submit Data in SHARE v1 Format
+    Please note that this endpoint is to ease the transition from SHARE v1 to SHARE v2 and sources
+    are encouraged to transition to submitting metadata in the SHARE v2 format.
+
+    Submitting data through the normalizeddata endpoint is strongly preferred as support for
+    the v1 format will not be continued.
+
+    v1 Format
+
+        For the full format please see https://github.com/erinspace/shareregistration/blob/master/push_endpoint/schemas.py
+
+        Required Fields: [
+            "title",
+            "contributors",
+            "uris",
+            "providerUpdatedDateTime"
+        ],
+
+    Create
+
+        Method:        POST
+        Body (JSON): {
+                        {
+                            "publisher":{
+                                "name": <publisher name>,
+                                "uri": <publisher uri>
+                            },
+                            "description": <description>,
+                            "contributors":[
+                                {
+                                    "name":<contributor name>,
+                                    "email": <email>,
+                                    "sameAs": <uri>
+                                },
+                                {
+                                    "name":<contributor name>
+                                }
+                            ],
+                            "title": <title>,
+                            "tags":[
+                                <tag>,
+                                <tag>
+                            ],
+                            "languages":[
+                                <language>
+                            ],
+                            "providerUpdatedDateTime": <time submitted>,
+                            "uris": {
+                                "canonicalUri": <uri>,
+                                "providerUris":[
+                                    <uri>
+                                ]
+                            }
+                        }
+
+                    }
+        Success:       200 OK
+    """
+    permission_classes = [ReadOnlyOrTokenHasScopeOrIsAuthenticated, ]
+    serializer_class = NormalizedDataSerializer
+
+    def get_queryset(self):
+        return NormalizedData.objects.all()
+
+    def post(self, request, *args, **kwargs):
+
+        prelim_data = request.data
+        prelim_data['source'] = ShareUser.objects.get(id=request.user.id)
```

Comment:
I don't think you have to put the user in `prelim_data`, you can just pass it to `store_data` separately.